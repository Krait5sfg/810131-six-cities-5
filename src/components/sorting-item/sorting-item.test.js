import React from 'react';
import renderer from 'react-test-renderer';
import SortingItem from './sorting-item';

describe(`SortingItem render correctly`, () => {
  it(`SortingItem open`, () => {
    const tree = renderer
      .create(<SortingItem
        itemName={`Popular`}
        isActive={true}
        onSortingItemClick={() => {}}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`SortingItem close`, () => {
    const tree = renderer
      .create(<SortingItem
        itemName={`Popular`}
        isActive={false}
        onSortingItemClick={() => {}}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
