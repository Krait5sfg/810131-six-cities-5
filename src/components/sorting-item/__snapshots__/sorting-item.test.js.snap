// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingItem render correctly SortingItem active 1`] = `
<li
  className="places__option places__option--active"
  data-sorting="Popular"
  onClick={[Function]}
  tabIndex="0"
>
  Popular
</li>
`;

exports[`SortingItem render correctly SortingItem not active 1`] = `
<li
  className="places__option "
  data-sorting="Popular"
  onClick={[Function]}
  tabIndex="0"
>
  Popular
</li>
`;
